massa-db-worker/src/massa_db.rs

```diff
@@ -851,6 +892,16 @@ impl MassaDBController for RawMassaDB<Slot, SlotSerializer, SlotDeserializer> {
         self.get_versioning_batch_to_stream(last_versioning_step, last_change_id)
     }
 
+    fn get_change_history_sizes(&self) -> (usize, usize) {
```

Comment:
maybe this do the same : 

fn get_change_history_sizes(&self) -> (usize, usize) {
    self.change_history_sizes.values()
        .fold((0, 0), |(acc_state, acc_version), &(state, version)| 
            (acc_state + state, acc_version + version))
}